frontend/packages/kubevirt-plugin/src/components/constants.js

```diff
@@ -0,0 +1 @@
+export const DASHES = '---';
```

Comment:
this should probably go to `console-shared`

Reply:
We use a single dash, `-`, throughout console. Is there a reason it's `---` here?

Reply:
IIRC, this was requested during UX review some time ago.
But I agree, it makes sense to unify and potentially change later everywhere.

Reply:
One more change: the `DASHES` are imported from `kubevirt-web-ui-components` now, so triple dash `---` is still used per former UXD review. This way it will be consistent across Virtual Machines pages and can be unified with OpenShift when moving `kubevirt-web-ui-components` under `console`. Or if it is a blocker, I can fix there and fire new release. @spadgett , are you ok with that, please?

Reply:
Sorry, I disagree. We should change it everywhere in console or not at all. It sounds like we reviewed one page and didn't consider any of the existing pages that already have a dash. IMO, the right thing to do is fix kubevrit-web-ui-components so that it's consistent with existing console pages.

We're adding kubevirt as a plugin is so that it feels like an integrated part of console. This might seem like a little thing, but it undercuts that goal. The kubevirt pages should look and behave like other console pages.

For the record, I prefer a single dash `-`. But I'm most concerned about keeping consistency.

Reply:
Can't we simply use the shared package here?

```ts
// packages/console-shared/src/constants/index.ts
export const DASH = '-';
```